tasks/clean-build-service.yml

```
@@ -0,0 +1,59 @@
+---
+- name: Check if systemd timer definition is present
+  stat:
+    path: '/etc/systemd/system/build-{{ beacon_node_service_name}}.timer'
+  register: systemd_service_build_timer
+
+- name: Check if the service is running
+  systemd_service:
+    name: 'build-{{ beacon_node_service_name }}.timer'
+    state: 'stopped'
```

You could disable it here too:
```suggestion
    state: 'stopped'
    enabled: false
```